### PR TITLE
Vibhi

### PR DESCRIPTION
**Pull request by vvaliapa**
Hi Anne,
I have updated the login screens for the docker.
Kindly review the updates.
Regards,
Vibhinna

### CODE REVIEW

File: labs/docker-101/1.md [hunk 1]
```diff
@@ -16,58 +16,73 @@ This is a hands-on lab, but each example builds on the previous and explains *wh
 
 ## Content Notice
 
-Thanks to the Cisco Live EU 2017 workshop "Introduction to Containers" for portions of this learning lab. Further reading is available in the excellent docker introduction at [imapex.io](https://github.com/imapex-training/mod_adv_docker/blob/master/README.md)
+Thanks to the Cisco Live EU 2017 workshop "Introduction to Containers" for portions of this Learning Lab. Further reading is available in the excellent docker introduction at [imapex.io](https://github.com/imapex-training/mod_adv_docker/blob/master/README.md).
 
 ## Installing Docker
+The Docker engine runs and manages containers on a host machine. The Docker client tools control the Docker engine. Both the Docker engine and the Docker client tools are now distributed by default as part of many Linux distributions. The engine and client tools are also available for download directly from [Docker.com](http://www.docker.com).
```
Comment (annegentle) [inline labs/docker-101/1.md:29]: Also mention that having a Docker ID is a prerequisite to this Lab. Maybe here or maybe in a pre-reqs section?

**Review by annegentle — CHANGES_REQUESTED** [review]
Looks good - just one additional pre-requisite please.

File: labs/docker-101/1.md (hunk 1 above)
Comment (vvaliapa) [reply labs/docker-101/1.md:29]: I have added a prerequisite section and updated your comments.

Comment (annegentle) [reply labs/docker-101/1.md:29]: Looks good! 👍

**Review by annegentle — APPROVED** [review]
Ship it 🚢

**Pull request merged**